multiqc/modules/pychopper/pychopper.py

```diff
@@ -0,0 +1,147 @@
+            'suffix' : '%',
+            'max': 100,
+            'min': 0,
+        }
+        
+        self.general_stats_addcols(data_general_stats, headers)
+
+        # Write data file
+        self.write_data_file(self.pychopper_data, 'multiqc_pychopper')
+
+        # Report sections
+        self.add_section (
+            name = "cDNA Read Classification",
+            description = (
+                """
+                This plot shows the cDNA read categories identified by Pychopper </br>
+                """
+            ),
+            helptext = (
+                """
+                There are three possible cases:
+
+                * **Primers found**: Full length cDNA reads with correct primers at both ends.
+                * **Rescued reads**: Split fusion reads.
+                * **Unusable**: Reads without correct primer combinations.
+                """
+            ),
+            anchor = 'pychopper_classification',
+            plot = self.plot_classification()
+        )
+        self.add_section (
+            name = "cDNA Strand Orientation",
+            description = (
+                """
+                This plot shows the strand orientation of full length cDNA reads
+                """
+            ),
+            helptext = (
+                """
+                Nanopore cDNA reads are always read forward. To estimate their original strand, 
+                Pychopper searches for the location of the start and end primers and assigns the reads accordingly.
+                """
+            ),
+            anchor = 'pychopper_orientation',
+            plot = self.plot_orientation()
+        )
+
+    # Plotting functions
+    def plot_classification(self):
+        """ Generate the cDNA read classification plot """
+
+        pconfig = {
+            'id': 'pychopper_classification_plot',
+            'title': 'Pychopper: Read classification',
+            'ylab': '',
+            'xDecimals': False,
+            'ymin': 0
+        }
+
+        data_classification = {}
+        for sample in self.pychopper_data.keys():
+            data_classification[sample] = {}
+            data_classification[sample]=self.pychopper_data[sample]['Classification']
+        
+        cats = ['Primers_found', 'Rescue', 'Unusable']
+        return bargraph.plot(data_classification, cats, pconfig)
+    
+    def plot_orientation(self):
+        """ Generate the read strand orientation plot """
+
+        pconfig = {
+            'id': 'pychopper_orientation_plot',
+            'title': 'Pychopper: Strand Orientation',
+            'ylab': '',
+            'cpswitch_c_active': False,  
+            'xDecimals': False,
+            'ymin': 0
+        }
+
+        data_orientation = {}
+        for sample in self.pychopper_data.keys():
+            data_orientation[sample] = {}
+            data_orientation[sample]=self.pychopper_data[sample]['Strand']
+
+        cats = ['+', '-']
```

Comment:
I think that this is a _little_ too minimalistic, so worth adding the word `strand` to make the categories a little clearer.

```suggestion
        cats = ['+ strand', '- strand']
```